packages/cli/src/commands/init.ts

```diff
@@ -120,6 +146,33 @@ function generateSchemasFrom(ast: AST, folder: string): void {
     );
     sourceFile.saveSync();
   }
+
+  const metricsPluginList = await getMetricPluginList(folder);
+  const imports =
+    metricsPluginList.length === 1
+      ? metricsPluginList[0]
+      : '\n  ' + metricsPluginList.join(',\n  ') + '\n';
+
+  const instantiations =
+    metricsPluginList.length === 1
+      ? `new ${metricsPluginList[0]}()`
+      : '\n' +
+        metricsPluginList
+          .map((input: string) => `    new ${input}()`)
+          .join(',\n') +
+        '\n  ';
+
+  const content = syftFileContent
+    .replace(/\{ IMPORT }/, imports)
+    .replace(/\{ INSTANTIATION }/, instantiations);
+  const syftSrcFile = project.createSourceFile(
+    path.join(folder, '/src/syft.ts'),
```

Comment:
looks like you are creating the file under "@syftdata/client" project. `folder` point to that i believe. did you test the location of syft.ts file after running the cli ?